src/Translator/TranslatableMessageInterface.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Validator\Translator;
+
+/**
+ * This interface provide methods for extract the translatable parts of a message.
+ */
+interface TranslatableMessageInterface
+{
+    /**
+     * Returns the message to translate.
+     *
+     * The message may include placeholders for message variable interpolation.
+     *
+     * @return string
+     */
+    public function getMessageTemplate();
```

Comment:
This method name may return two types of message.

* Messages with placeholders
* Messages without placeholders.